lib/App/Rakubrew/Build.pm

```diff
@@ -187,16 +187,25 @@ sub build_triple {
 
 sub build_zef {
     my $version = shift;
+    my $zef_version = shift;
 
     _check_git();
-
     chdir catdir($versions_dir, $version);
     unless (-d 'zef') {
         run "$GIT clone $git_repos{zef}";
     }
     chdir 'zef';
-    run "$GIT pull -q";
```

Comment:
Now the pull got lost. This prevents updates and installing tags later than the original clone.

Reply:
The problem with the pull is that it errors. I'll have a look anyway.